src/components/Table/index.js

```diff
@@ -201,7 +206,33 @@ const Table = () => {
             },
           },
           {
-            Header: 'Questions',
+            Header: () => {
+              const randomQuestion = () => {
+                const random = Math.floor(Math.random() * questions.length);
+                const questionId = questions[random].id;
+                const questionSlug = questions[questionId].url;
```

Comment:
Were you able to rebase off the `master` branch? `url` is no longer used there and if we were to land this PR now, it wouldn't work! To rebase means to pull in the latest commits from the branch we're targeting, ie. `master`, and then apply all of your commits in the order you made them. It can seem tricky if it's your first time doing it - let me know if you'd like for me to walk you through it 👍🏽

Once you've rebased..

```
const questionSlug = questions[questionId].url;
```

will need to be changed to..

```
const questionSlug = `https://leetcode.com/problems/${questions[questionId].slug}/`;
```

See questions.json here for how the data is presented - we no longer store the `https://leetcode.com/problems/` portion since we know that is "_static_" in a sense - we only store the unique identifying part of the address, aka a "_slug_" ([MDN docs](https://developer.mozilla.org/en-US/docs/Glossary/Slug)).

Reply:
@seanprashad  Yes, I am doing this for the first time, so can you please assist me😅

Reply:
![image](https://user-images.githubusercontent.com/73271539/185803252-9377ac71-29a0-4e1b-9f1c-5d209293b3f6.png)

Reply:
I typed these commands till now, I am not sure what to do after this..

Reply:
PS: I figured it out, I was able to successfully rebase the master branch, and new changes are now present in the master branch.

Reply:
Hmm, seems like, after adding these changes..
```
const questionSlug = questions[questionId].slug; 
window.open(`https://leetcode.com/problems/${questionSlug}/`, '_blank');
```
There is a bug, the `questionSlug` is `undefined`, because of which leetcode is not able to find the page.
![image](https://user-images.githubusercontent.com/73271539/185804443-b0cf8cdb-79dd-49d2-9ce9-0abdd4f934fb.png)

Reply:
I think you needed to pull from a different remote, ie. the `master` branch from `seanprashad/leetcode-patterns` instead of `master` from `Shashank5665/leetcode-patterns`.

We can see the remote repos you have via `git remote -v`:

```
 ✘ sprashad@home  ~/projects/leetcode-patterns   randomQuestion  git remote -v
origin  git@github.com:seanprashad/leetcode-patterns.git (fetch)
origin  git@github.com:seanprashad/leetcode-patterns.git (push)
```

As you can see above, I only have remote for the original repo (_ie. the one you're submitting a PR to_). You'll need to run the following to add mine as an "_upstream_" remote:

```
// This is for SSH - you might use HTTPS
git remote add upstream git@github.com:seanprashad/leetcode-patterns.git
```

Afterwards, you can verify we've correctly added my repo as a remote via `git remote -v`.

I think you can then run the following to pull the latest `master` branch from `seanprashad/leetcode-patterns` (_which has the `slug` changes_):

```
git checkout master
git pull upstream master
git checkout -
git rebase master
git push origin -f
```

Note that we'll need to force push after rebasing - this is because commit hashes change after rebasing, even though the contents of our commit doesn't!

Reply:
Hello @seanprashad, done with all the steps that you have mentioned, it took me a while to know what's going on, but surfing google a lil bit and with the help of your instructions, I have done it. Now, after starting the server, the feature works very well😁( with the slug inclusion ). I think everything is in place, So, is it just a pull request now?

Reply:
Awesome! Please do mark it as a PR and I'll review it when I get a chance!

Reply:
Sure @seanprashad !!